test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -416,13 +334,15 @@ TEST_F(ToriiServiceTest, CheckHash) {
     tx_hashes.push_back(iroha::hash(*tx).to_string());
   }
 
+  auto client = torii::CommandSyncClient(Ip, Port);
+
   // get statuses of transactions
   for (auto &hash : tx_hashes) {
     iroha::protocol::TxStatusRequest tx_request;
```

Comment:
Use shared_model instead.

Reply:
I just added reusable client, I did not touch other parts of the test